SalesforceCommerceCloud/pwa-kit
# Allow ssr cookies

# Description

SSR currently blocks setting cookies and strips them from requests because our CloudFront doesn't pass cookies. We are adding an environment setting to allow it in CloudFront and this allows the setting to disable the cookie blocking code from here.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Put setting in SSR code to disable cookie blocking
- Add test endpoint to MRT app for cookies

# How to Test-Drive This PR

- Locally set MRT_ALLOW_COOKIES to true in env or configure localAllowCookies to true and then see that cookies work
- Use /cookie?name=test&value=test in mrt app to check

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
